hackytrack/syntax/rename.py

```diff
@@ -0,0 +1,9 @@
+class Rename(object):
+    def __init__(self, record_reader, command_parts):
+        self.command_parts = command_parts
+        self.record_reader = record_reader
+
+    def process(self):
+        index = int(self.command_parts[0])
```

Comment:
I mean, I guess you might want to put a negative number in here, but it might not be something we want to allow. Also, this might explode if an index that doesn't exist in the record, or if the number entered is larger than the max int.